vpr/src/pack/greedy_seed_selector.cpp

```diff
@@ -0,0 +1,237 @@
+/**
+ * @file
+ * @author  Alex Singer
+ * @date    November 2024
+ * @brief   The definitions of the Greedy Seed Selector class.
+ */
+
+#include "greedy_seed_selector.h"
+
+#include <algorithm>
+#include "atom_netlist.h"
+#include "cluster_legalizer.h"
+#include "cluster_util.h"
+#include "echo_files.h"
+#include "prepack.h"
+#include "vpr_error.h"
+#include "vpr_types.h"
+#include "vtr_assert.h"
+#include "vtr_math.h"
+#include "vtr_vector.h"
+
+/**
+ * @brief Helper method that computes the seed gain of the give atom block.
```

Comment:
give -> given
A higher gain means it is a more desirable seed (starting point) for a new cluster.